Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM golang:alpine AS build
+
+RUN apk add --no-cache gcc libc-dev git
```

Comment:
Do we need these packages, if not let's remove them. Hard to remove once we have added something :thinking: